modules/styling-transform/lib/utils/parseObjectToStaticValue.ts

```diff
@@ -105,6 +105,15 @@ function parsePropertyToStaticValue(node: ts.Node, context: TransformerContext):
     }
   }
 
+  // { name }
+  if (ts.isShorthandPropertyAssignment(node)) {
```

Comment:
The name of this function is clear enough, but I can't find any docs on this function. Bizarre.

Reply:
https://ts-ast-viewer.com/#code/MYewdgzgLgBAZiEMC8MDkCRoNwCheiSwBGAhgE4owDeuM98iMA9MzAMoAWI5UnpYACYAFciAAOAU14BPAIIQIASwDmYALaSwUXAF8gA

Reply:
There is almost no documentation on any TypeScript AST or Type Checker functions. There's no JSDoc. The source code is all I get: https://github.com/microsoft/TypeScript